src/components/ContentHighlights/HighlightStepper/ContentConfirmContentCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Delete } from '@edx/paragon/icons';
+import { IconButton, Icon } from '@edx/paragon';
+import { connect } from 'react-redux';
+import ContentHighlightCardItem from '../ContentHighlightCardItem';
+import { useContentHighlightsContext } from '../data/hooks';
+import { generateAboutPageUrl } from '../data/constants';
+
+const ContentConfirmContentCard = ({ enterpriseSlug, original }) => {
+  const { deleteSelectedRowId } = useContentHighlightsContext();
+  const [deleteKey, setDeleteKey] = useState(null);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const {
+    title,
+    contentType,
+    partners,
+    cardImageUrl,
+    originalImageUrl,
+    firstEnrollablePaidSeatPrice,
+    aggregationKey,
+    key,
+  } = original;
+  // eslint-disable-next-line no-shadow
+  useEffect(() => {
+    if (deleteKey === aggregationKey) {
```

Comment:
[clarification] Why is the `useEffect` necessary vs. using a callback function? E.g.

```jsx
<IconButton
  invertColors
  isActive
  src={Delete}
  iconAs={Icon}
  alt="Delete"
  onClick={() => deleteSelectedRowId(aggregationKey)}
  className="ml-1 flex-shrink-0"
/>
```

Reply:
I believe bad implementation of other parts of the code rendering the cards after it was deleted from the past required me to go down this route initially, which is now totally unnecessary. 

All suggested fixes regarding this page have been implemented and appears to be working correctly